io/disk/disktest.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+# Copyright: 2016 IBM
+# Author: Nosheen Pathan <nopathan@linux.vnet.ibm.com>
+# Based on code by
+# Author: Martin Bligh (mbligh@google.com)
+# Copyright: 2007 Google, Inc.
```

Comment:
Please add the original location, it helps with review as well as when looking at some bugs (which might be present/missing in the original file)

```
https://github.com/autotest/autotest-client-tests/tree/master/disktest
```